libarchive/archive_write_add_filter_ab.c

```diff
@@ -0,0 +1,372 @@
+/*-
+ * Copyright (c) 2016 dosomder
```

Comment:
Please don't use nicknames in the Copyright header. You can pass the copyright to someone else if you want.